src/sparse/KokkosSparse_gauss_seidel_handle.hpp

```diff
@@ -564,32 +564,33 @@ namespace KokkosSparse{
     
     bool use_teams() const
     {
+      bool return_value = false;
 #if defined( KOKKOS_ENABLE_SERIAL )
       if (Kokkos::Impl::is_same< Kokkos::Serial , ExecutionSpace >::value) {
```

Comment:
We should at some point replace `Kokkos::Impl::is_same` with `std::is_same` (which is in the `<type_traits>` header).